services/bugmon-win/setup.sh

```diff
@@ -0,0 +1,98 @@
+shopt -s extglob
+
+retry () { i=0; while [ "$i" -lt 9 ]; do if "$@"; then return; else sleep 30; fi; i="$((i+1))"; done; "$@"; }
+retry_curl () { curl -sSL --compressed --connect-timeout 25 --fail --retry 5 -w "%{stderr}[downloaded %{url_effective}]\n" "$@"; }
+
+# base msys packages
+retry pacman --noconfirm -S \
+  mingw-w64-x86_64-curl \
+  openssh \
+  p7zip \
+  patch \
+  psmisc \
+  subversion \
+  tar \
+  zstd
+pacman --noconfirm -Scc
+killall -TERM gpg-agent || true
+pacman --noconfirm -Rs psmisc
+
+# get nuget
+retry_curl "https://aka.ms/nugetclidl" -o msys64/usr/bin/nuget.exe
+
+# get python
+VER=3.10.8
+retry nuget install python -ExcludeVersion -OutputDirectory . -Version "$VER"
+rm -rf msys64/opt/python
+mkdir -p msys64/opt
+mv python/tools msys64/opt/python
+rm -rf python
+PATH="$PWD/msys64/opt/python:$PWD/msys64/opt/python/Scripts:$PATH"
+which python
+python -V
+
+# patch pip to workaround https://github.com/pypa/pip/issues/4368
+sed -i "s/^\\(    \\)maker = PipScriptMaker(.*/&\r\n\\1maker.executable = '\\/usr\\/bin\\/env python'/" \
+  msys64/opt/python/Lib/site-packages/pip/_internal/operations/install/wheel.py
+
+# configure pip
+mkdir -p pip
+cat << EOF > pip/pip.ini
+[global]
+disable-pip-version-check = true
+no-cache-dir = false
+
+[list]
+format = columns
+
+[install]
+upgrade-strategy = only-if-needed
+progress-bar = off
+EOF
+
+# force-upgrade pip to include the above patch
+# have to use `python -m pip` until PATH is updated externally
+# otherwise /usr/bin/env will select the old `pip` in mozbuild
+retry python -m pip install --upgrade --force-reinstall pip
+
+# patch new pip to workaround https://github.com/pypa/pip/issues/4368
+sed -i "s/^\\(    \\)maker = PipScriptMaker(.*/&\r\n\\1maker.executable = '\\/usr\\/bin\\/env python'/" \
+  msys64/opt/python/Lib/site-packages/pip/_internal/operations/install/wheel.py
+
+retry_curl -O https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.mozilla-central.latest.taskgraph.decision/artifacts/public/label-to-taskid.json
+resolve_tc () {
+python - "$1" << EOF
+import json
+import sys
+with open("label-to-taskid.json") as fd:
+  label_to_task = json.load(fd)
+task_id = label_to_task[f"toolchain-win64-{sys.argv[1]}"]
+print(f"https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/{task_id}/artifacts/public/build/{sys.argv[1]}.tar.zst")
+EOF
+}
+
+# get new minidump-stackwalk
+retry_curl -O "$(resolve_tc minidump-stackwalk)"
+zstdcat minidump-stackwalk.tar.zst | tar xv
+mv minidump-stackwalk/minidump-stackwalk.exe msys64/usr/bin/
+rm -rf minidump-stackwalk minidump-stackwalk.tar.zst
+./msys64/usr/bin/minidump-stackwalk.exe --version
+
+rm label-to-taskid.json
+
+# install utils to match linux ci images
+retry python -m pip install \
+  psutil \
+  virtualenv
+
+rm -rf \
+  msys64/mingw64/share/doc/ \
+  msys64/mingw64/share/info/ \
+  msys64/mingw64/share/man/ \
+  msys64/usr/share/doc/ \
+  msys64/usr/share/info/ \
+  msys64/usr/share/man/
+
+
```

Comment:
You're missing the part where the launch script is copied and the archive is actually created. See [grizzly-win/setup.sh](https://github.com/MozillaSecurity/orion/blob/8c105cd1285431499de0f1ce10c029971ef2c9c8/services/grizzly-win/setup.sh#L124-L128).

Reply:
Fixed in [2ec330b](https://github.com/MozillaSecurity/orion/pull/360/commits/2ec330bb08c297fce84c108ed26eade8cc045916).